data/en/footer.yml

```diff
@@ -8,10 +8,15 @@ footer:
     product_items:
     - name: Ensign
       link: https://rotational.app
-    - name: Documentation
+    - name: Pricing
+      link: /ensign-pricing
+    - name: Docs
```

Comment:
`Documentation` looked really long after adding `Pricing`. Shortening the word to `Docs` makes it more in line with the others and it's common to see `Docs` used on other sites.

Reply:
Makes sense to me!